guides/kubernetes/kubernetes.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes
+
+Kubernates(also written as K8s) is an open source system for automatic , scaling and managemnt of [containerized applications](https://searchitoperations.techtarget.com/definition/application-containerization-app-containerization). It is the most popular tool among all currently present which is used mostly by all the major giants of the industry including Google who actually founded it, Amazon and Microsoft etc.
+
+The Github repository of kubernates project can be visited from [here](https://github.com/kubernetes/kubernetes)
+
+Before knowing it further you must be aware of what actually a docker is,what are containers and containerized applications. 
+
+## Why kubernetes ?
+
+As i mentioned that kubernates is a product of google which was released in July 2015 after a decade and half year effort of google who was earlier facing with its own internal Borg system which had certain drawbacks. So they thought of creating something new which can be standarized which can be used by all and the result is kubernetes.
+
+If you are aware of [docker compose](https://docs.docker.com/compose/) then you might also be aware that docker compose can't be put into production .It is a config file which is helpful in running all containers together but suppose that if it is used into production and if any container stops working (crashes for any reason) then the whole application stops and it becomes a tedious task to restart. There were few other problems which can't be solved by docker compose  such as maintaing logs ,scaling,monitoring etc
+
+## Need of Orchestration Engine 
+
+The work of orchestration engine starts from deployment of applications.There are various things that are taken care by it which can not be in case of docker compose. These can be
+
+ * Storage domain 
+ * Compute domain
+ * Application provisioning
+ * Monitoring 
+ * Lifecycle Management
+
+Kubernetes is one of the popular orchestration engine.Inspite of [docker swarm](https://docs.docker.com/get-started/part4/) which is a native cluster of docker , it has started releasing native support for kubernetes as well.
+kubernetes provides an infrastructure to build a truly docker centric application environment.
+
+## Some Cool features of kubernetes
+
+These can be seen as
+
+* Portable --> It is easy to migrate from one cloud to another by just changing the kubernates configuration so no need to write complete deoloyment setup again
```

Comment:
1) Spell check of "kubernates" to "kubernetes".
2) Spell check of "deoloyment" to "deployment".